src/assets/i18n/fr.ts

```diff
@@ -1,528 +1,528 @@
 export const fr = {
     "common":{
-        "followers":"Followers",
-        "allempty":"All Empty",
-        "releaseToRefresh":"Release to refresh",
-        "beingRefresh":"Refreshing...",
-        "cancel":"Cancel",
+        "followers":"Abonnés",
+        "allempty":"Tout est vide",
+        "releaseToRefresh":"Appuyer pour actualiser",
+        "beingRefresh":"Actualisation en cours",
+        "cancel":"Annuler",
         "ok":"OK",
-        "confirm":"Confirm",
-        "des":"Clear <strong>all cached data</strong>!!!",
-        "prompt":"Prompt",
-        "des1":"Confirm New Feeds",
-        "server":"Feeds Publisher Account:",
+        "confirm":"Confirmer",
+        "des":"Supprimer <strong>toutes les données en cache</strong>!!!",
```

Comment:
toutes les données temporaires

Reply:
Still could be changed? we can integrate it until all basically are settled.